### DISCUSSION TITLE
广州相关地铁之最资料扩充

### DISCUSSION THREAD

**Discussion by 1889737514** [Ivysauro/CNRT#42] [Ideas]
1. 广州18号线开通后地铁最长区间和最长地下区间应为番禺广场至横沥区间（区间长25806m）
2. 目前数字编号最大的运营线路为广州22号线
3. 广州18号线和22号线与北京大兴机场线同为160km/h的设计时速
4. 最多换乘车站增加广州南站（四线六向，2单向、7双向、22双向、佛山2单向，虽然佛山2目前是出站换乘状态）

**Comment by 1889737514** [comment]
补充：广州地铁18号线于2021年9月28日开通，佛山地铁2号线于2021年12月28日开通，广州地铁22号线于2022年3月31日开通，广州地铁7号线西延段（跨市）于2022年5月1日开通

**Reply by 1889737514** [reply]
另：佛山地铁2号线可以加入到跨城连通一栏

**Reply by Ivysauro** [reply]
感谢提供最新信息🤓

**Comment by 1889737514** [comment]
另：南海有轨电车一号线现为脱网状态（2021年8月18日开通首通段并脱网，该线本质为轻轨，采用广州&佛山地铁计价方式，将于今年并入线网（在林岳西站与佛山地铁2号线同台换乘））

**Comment by Ivysauro** [comment]
有兴趣可以直接修改提交pr😉

**Reply by 1889737514** [reply]
那我试一下（

**Comment by Ivysauro** [comment]
pr已处理 辛苦了

**Comment by chenw23** [comment]
复核确认：本讨论列出的广州18/22号线及相关“之最”已在历史 PR #43 中处理，并已存在于当前数据：番禺广场—横沥最长站间距、18/22号线160km/h设计时速、22号线为数字编号最大的运营线路、广州南站的换乘统计等。因此无需重复修改。

对应历史 PR：https://github.com/Ivysauro/CNRT/pull/43
